ShopGo/LoginScreen/LoginSignupExtenstion.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  LoginSignupExtenstion.swift
+//  ShopGo
+//
+//  Created by Quratulain on 7/26/23.
+//
+
+import UIKit
+
+extension LoginSignupViewController {
+    // function for radius of login button inside the view
+    func buttonCornerStyle() {
+        loginButtonOutlet.layer.cornerRadius = 22
+        loginButtonOutlet.layer.masksToBounds = true
```

Comment:
do we need this method? We already have similar stuff in `CustomButton.swift` can we not reuse that?

Reply:
I'll create a separate class for this too

Reply:
fixed